### PR TITLE
Modified event query to only return future or ongoing events

### PR DESCRIPTION
**Pull request by israeltq**

### CODE REVIEW

File: api/handlers/event.js [hunk 1]
```diff
@@ -265,10 +265,12 @@ let findEvents = async function(req, res) {
             return res.status(404).send("Requesting user not found");
         }
 
-        let nearEvents = await Event.find(nearEventsQuery);
+        let nearEvents = await Event.find(nearEventsQuery)
+                                    .gte("endTime", (Date.now() / 1000));
```
Comment (adrianosela) [inline api/handlers/event.js]: indentation

Comment (israeltq) [reply api/handlers/event.js]: Yes, looks like codacy doesnt like this. I am currently looking for the standard in javascript chaining functions

**Comment by israeltq** [comment]
> 
> 
> Easy fix haha, is this all the places we call Event.find?

That's a good point. We also use them for retrieving user created, pending, and subscribed events. @chrisneuf do you think these calls also need it? Maybe not created, I am not sure

**Comment by chrisneuf** [comment]
> > Easy fix haha, is this all the places we call Event.find?
> 
> That's a good point. We also use them for retrieving user created, pending, and subscribed events. @chrisneuf do you think these calls also need it? Maybe not created, I am not sure

Yeah we had talked about it in the chat and decided it made sense to include it everywhere so the user doesn't get cluttered with old events

**Comment by israeltq** [comment]
> 
> 
> > > Easy fix haha, is this all the places we call Event.find?
> > 
> > 
> > That's a good point. We also use them for retrieving user created, pending, and subscribed events. @chrisneuf do you think these calls also need it? Maybe not created, I am not sure
> 
> Yeah we had talked about it in the chat and decided it made sense to include it everywhere so the user doesn't get cluttered with old events

Okok, will add to the other calls too. Thanks

**Pull request merged**